### DISCUSSION TITLE
wow !

### DISCUSSION THREAD

**Discussion by DoctorAcc22** [GitzJoey/DCSLab#235] [Show and tell]
![emoticon-thumb-up](https://user-images.githubusercontent.com/105140626/173150514-a96c66fa-f3f0-421e-b1b7-1139ebd6fc91.jpg)

**Comment by boylucnam** [comment]
Hi! when i install the project and run the project but on test registration i get this error please tell me how to fix it. Thank!
![image](https://user-images.githubusercontent.com/104551491/174514820-60336332-c19c-499e-9377-dd8be2c83234.png)

**Reply by GitzJoey** [reply]
@boylucnam 
make sure u follow the instructions in readme

**Reply by HugoYaw22** [reply]
hey @boylucnam have you successfully done the installation?
